dss-api.yml

```diff
@@ -660,6 +660,258 @@ paths:
                     enum: [unhandled_exception, Forbidden, OAuthResponseProblem, not_found]
                 required:
                   - code
+  /collections:
+    put:
+      summary: Create a collection.
+      description: |
+        Create a new collection.
+
+        Collections are sets of links to files, bundles, other collections, or fragments of JSON metadata files. Each
+        entry in the input set of links is checked for referential integrity (the link target must exist in the replica
+        referenced). Up to 1000 items can be referenced in a new collection, or added or removed using
```

Comment:
Nit: My reading from the later API specification is that 1000 items can be added and 1000 items can be removed.  My reading of this text here suggests that 1000 can be added or removed.

Reply:
I agree there's a bit of ambiguity, but not sure how to re-word this better. Any suggestions?